node/12-user/Dockerfile

```diff
@@ -22,6 +22,22 @@ RUN set -ex; \
     ; \
   rm -rf /var/lib/apt/lists/*
 
+# Install docker
+RUN apt-get update && apt-get install -y \
```

Comment:
Any reason to not lump this in with the `apt-get update ...` stuff above?  Seems weird to do this twice.

Reply:
I thought I'd lump dependences with the commands that require them. But it does HIT existing dependences again and adds time. I'll converge the updates in one run